### PR TITLE
Adds pagination to apps view page

### PR DESCRIPTION
**Pull request by elijahbenizzy**
This is not perfect -- but it'll do. The caveat is that parent/child pointers aren't always found -- we need to add that to the API.

[Short description explaining the high-level reason for the pull request]

## Checklist

- [ ] PR has an informative and human-readable title (this will be pulled into the release notes)
- [ ] Changes are limited to a single goal (no scope creep)
- [ ] Code passed the pre-commit check & code is left cleaner/nicer than when first encountered.
- [ ] Any _change_ in functionality is tested
- [ ] New functions are documented (with a description, list of inputs, and expected output)
- [ ] Placeholder code is flagged / future TODOs are captured in comments
- [ ] Project documentation has been updated if adding/changing functionality.

### CODE REVIEW

**Comment by skrawcz** [comment]
@elijahbenizzy node.js error

**Comment by elijahbenizzy** [comment]
#342

**Comment by skrawcz** [comment]
> The caveat is that parent/child pointers aren't always found -- we need to add that to the API.

New issue?

**Comment by elijahbenizzy** [comment]
image

File: burr/tracking/server/backend.py [hunk 1]
```diff
@@ -230,8 +235,13 @@ async def _load_metadata(self, metadata_path: str) -> models.ApplicationMetadata
...
     async def list_apps(
-        self, request: fastapi.Request, project_id: str, partition_key: Optional[str]
-    ) -> Sequence[ApplicationSummary]:
+        self,
+        request: fastapi.Request,
+        project_id: str,
+        partition_key: Optional[str],
+        limit: int = 100,
+        offset: int = 0,
+    ) -> Tuple[Sequence[ApplicationSummary], bool]:
```
Comment (skrawcz) [inline burr/tracking/server/backend.py:244]: bool? this should be int right?

File: burr/tracking/server/backend.py [hunk 2]
```diff
@@ -101,8 +101,13 @@ async def list_projects(self, request: fastapi.Request) -> Sequence[schema.Proje
...
     async def list_apps(
-        self, request: fastapi.Request, project_id: str, partition_key: Optional[str]
-    ) -> Sequence[schema.ApplicationSummary]:
+        self,
+        request: fastapi.Request,
+        project_id: str,
+        partition_key: Optional[str],
+        limit: int,
+        offset: int,
+    ) -> Tuple[Sequence[schema.ApplicationSummary], int]:
```
Comment (skrawcz) [inline burr/tracking/server/backend.py:110]: adjust doc string to say why int

**Comment by skrawcz** [comment]
> image

where is the pagination?

**Comment by elijahbenizzy** [comment]
image

One left todo -- add ... to show that we have more pages. In the code there's aline on where to do it.

**Review by skrawcz — APPROVED** [review]

**Pull request merged**